package.json

```diff
@@ -152,7 +154,11 @@
   },
   "husky": {
     "hooks": {
-      "pre-push": "echo 'running lint...' && npm run lint && echo 'running tests...' && npm run test"
+      "pre-commit": "lint-staged",
+      "pre-push": "echo 'running tests...' && npm run test"
     }
+  },
+  "lint-staged": {
+    "client/app/**/*.{js,jsx}": ["npm run lint-staged", "git add"]
```

Comment:
Not sure if `git add` is needed - `eslint` will not change anything without `--fix` option

Reply:
Right. Fixed.